### ISSUE TITLE
Detalhar o formato do documento

### ISSUE THREAD

**Issue by josenaves** [react-native-drops/react-native-faq#5]
Como será organizado o documento?

Quais seções teremos - Introdução, Ambiente de desenvolvimento, React Core, Styles, etc...

Usar um grande arquivo ou quebrar cada issue num arquivo separado ?

Definir estilo de formatação do documento.

**Comment by rafaellincoln** [comment]
Acho que seria legal usar o README.md como index e criar um sistema de pastas para as seções, e fazendo link entre as documentações, para evitar duplicar informações e evitar contradições.

Quanto as seções podemos ir criando de acordo com a necessidade, mas começar com uma estrutura padrão parecida com:

- Introdução
  - Instalação
    - Windowns.md
    - Linux.md
    - Mac.md
  - Dicas
- Styles
- Bridge
- Redux

Quanto ao estilo de formatação, podemos seguir a [documentação do JavaScript Style da Airbnb](https://github.com/airbnb/javascript).

**Comment by josenaves** [comment]
@rafaellincoln Só uma coisa - sobre estilo de formatação - estava me referindo ao documento em si (seções, h1, h2, etc)... e não a formatação de código (estilo).

**Comment by rafaellincoln** [comment]
@josenaves Sim, eu me referi a isso, a gente seguir o padrão (h1, h2, etc..) que eles utilizam na documentação, e não o javascript em si.
